### ISSUE TITLE
pydensecrf is a dependency!!

### ISSUE THREAD

**Issue by ayoosh007** [Snowad14/Manga-MTL-Website#1]
pydensecrf is a pain in the ass to install on windows and should be clearly mentioned in the requirements.txt

**Comment by Snowad14** [comment]
You're right, but this is a very old project and, given its uselessness, it shouldn't be installed by anyone.

**Comment by ayoosh007** [comment]
I was trying to build something similar to it on my own and tried finding
inspiration it's the only on I found.Btw I loved your website sad to see it
shutdown.
